libs/island-ui/core/src/lib/Footer/Footer.tsx

```diff
@@ -161,16 +143,15 @@ export const Footer = ({
                         </Link>
                       </Text>
                     </Inline>
-                  </Box>
-                </Box>
+                  </Inline>
+                </div>
               </Box>
             </GridColumn>
             {showMiddleLinks ? (
               <GridColumn
-                span={['12/12', '12/12', '6/12']}
+                span={['12/12', '12/12', '8/12', '9/12']}
                 paddingBottom={[4, 4, 0]}
-                paddingTop={[6, 6, 0]}
-                className={styles.withDecorator}
+                paddingTop={[4, 4, 0]}
               >
                 <Box paddingX={[0, 0, 1]}>
                   {middleLinksTitle ? (
```

Comment:
maybe we should try to stay consistent when there is only one render option? instead of returning null, might need to cast to boolean (!!)
```suggestion
                  {middleLinksTitle && (
```